eng/Versions.props

```diff
@@ -30,6 +30,7 @@
     <BasicUndoVersion>0.9.3</BasicUndoVersion>
     <BenchmarkDotNetVersion>0.11.1</BenchmarkDotNetVersion>
     <CommandLineParserVersion>2.0.273-beta</CommandLineParserVersion>
+    <DiffPlexVersion>1.4.3</DiffPlexVersion>
```

Comment:
We've got this recorded in whatever OSS tool we need to record this in?

Reply:
I have no idea /cc @tmat

Reply:
The component is now registered.